libcloud/dns/drivers/gandi.py

```diff
@@ -147,16 +147,20 @@ def delete_zone(self, zone):
         return res.object
 
     def _to_record(self, record, zone):
+        extras = {'ttl': record['ttl']}
+        value = record['value']
+        if record['type'] == 'MX':
+            extras['priority'] = record['value'].split(' ')[0]
+            value = record['value'].split(' ')[1]
```

Comment:
Just curious why you split the value here and take the second (and not first item).